Makefile

```diff
@@ -1,21 +1,19 @@
 CPP = emcc
-CPPFLAGS = -O3 -s TOTAL_MEMORY=318767104 -s NO_FILESYSTEM=1 -s NO_BROWSER=1 --closure 1 --llvm-lto 1 --pre-js enc/pre.js
+CPPFLAGS = -O3 -s TOTAL_MEMORY=318767104 -s NO_FILESYSTEM=1 --closure 1 --pre-js enc/pre.js
 
-COMMONDIR = vendor/brotli/common
-ENCDIR = vendor/brotli/enc
-ENCSRC = enc/encode.c $(wildcard $(COMMONDIR)/*.c) $(wildcard $(ENCDIR)/*.c)
+COMMONDIR = vendor/brotli/c/common
+ENCDIR = vendor/brotli/c/enc
+ENCSRC = compress.c enc/encode.c $(wildcard $(COMMONDIR)/*.c) $(wildcard $(ENCDIR)/*.c)
```

Comment:
guess we don't need encode.c anymore since the js wrapper always uses encodeWithDictionary?